package.json

```diff
@@ -14,7 +14,7 @@
     "lint:fix": "yarn lint --fix",
     "create-modular-react-app": "ts-node packages/create-modular-react-app/src/cli.ts",
     "modular": "ts-node packages/modular-scripts/src/cli.ts",
-    "test": "yarn modular test --regex --watchAll false --runInBand --env node",
+    "test": "cross-env NODE_OPTIONS=--max_old_space_size=4096 yarn modular test --regex --watchAll false --runInBand --env node --transformIgnorePatterns \"node_modules/(?!rollup)/\"",
```

Comment:
Using cross-env mostly so that it's compatible with Windows (compared to just using `NODE_OPTIONS="....`)